airgun/views/rhai.py

```diff
@@ -0,0 +1,22 @@
+from widgetastic_patternfly import Button
+from widgetastic.widget import Text
+
+from airgun.views.common import BaseLoggedInView
+
+
+class ActionsDetailsView(BaseLoggedInView):
+    title = Text(".//h1[normalize-space(.)='Actions']")
+    export_csv = Button("Export CSV")
+    stability_issues = Text(".//span[normalize-space(.)='Stability']"
+                            "/parent::span/following-sibling::span")
```

Comment:
Could you please share html snippet of that element? There should be simpler way to locate it

Reply:
```html
<a ng-class="category.id" ui-sref="app.topic({id: category.id, product: getSelectedProduct()})" href="/redhat_access/insights/actions/stability?product=all" class="stability">
    <i ng-class="category.id + '-color'" aria-hidden="true" class="fa fa-square inline-right stability-color"></i>
    <span translate="translate" class="desc">
        <span>Stability</span>
    </span>
    <span class="count">&nbsp;(1)</span>
</a>
```

Reply:
Here you go, simpler and independent from current user's locale:
```
.//a[@class='stability']/span[@class='count']
```
If you feel like relying on exact text is a must in such case (which i strongly discourage), then you can try:
```
.//a[span[normalize-space(.)='Stability']]/span[@class='count']
```